app/controllers/hub/efile_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Hub
+  class EfileController < Hub::BaseController
```

Comment:
I would name this something more specific to state file. i think it's a little confusing that the current pages for viewing efile-related things are in the `EfileSubmissionsController` while this is just `EfileController`. Consider either namespacing under a `StateFile` module or including state file in the name of the controller

Reply:
Done